tests/test_unified_llm_runtime.py

```
@@ -114,10 +120,14 @@ class _FakeTokenizer:
         eos_token_id = 0
         pad_token_id = 0
 
-        def __call__(self, prompt: str, return_tensors: str | None = None) -> _FakeEncoding:  # noqa: ARG002
+        def __call__(
+            self, prompt: str, return_tensors: str | None = None
```

Ruff (0.14.5)

Unused method argument: `return_tensors`

(ARG002)